plutus/budget_manager/budget_manager.py

```diff
@@ -0,0 +1,230 @@
+import click
+import datetime
+from DBUtils.PooledDB import PooledDB
+import logging
+import markus
+import os
+
+import pymysql
+import subprocess
+import sys
+from verify import verify_project_yaml, verify_parent_yaml, verify_labels_yaml, verify_default_yaml
+import yaml
+
+from google.cloud import billing_budgets_v1beta1
+from google.cloud import resource_manager
+from google.cloud import storage
+
+from google.api_core.exceptions import GoogleAPICallError, RetryError
+
+from constants import (
+    APP,
+    PLUTUS_CONFIG_TYPE_PROJECT,
+    PLUTUS_CONFIG_TYPE_PARENT,
+    PLUTUS_CONFIG_TYPE_LABEL,
+    PLUTUS_CONFIG_TYPE_DEFAULT
+)
+
+from mysql import upsert_budget
+from gcp_helper import GcpHelper
+from project_budget import ProjectBudget
+
+# TODO - CircleCI should push yaml config to GCS after master branch pushes
+# TODO - add sample config yaml file to repo
+
+log = logging.getLogger(APP)
+metrics = markus.get_metrics(APP)
+
+@click.command()
+@click.option('--gcs-bucket', default=None)
+@click.option('--gcs-file-path', default=None)
+@click.option('--billing-account-id', default=None)
+@click.option('--default-pubsub-topic', default='projects/moz-fx-billing-212017/topics/plutus-budget-notifications')
+@click.option('--mysql-host', default='localhost')
+@click.option('--mysql-port', default='3306')
+@click.option('--mysql-user', default='root')
+@click.option('--mysql-pass', default='secret')
+@click.option('--mysql-db', default='plutus')
+# Used for local testing without container, and without using gcs file
+@click.option('--local-mode', is_flag=True)
+@click.option('--dry-run', is_flag=True)
+def main(gcs_bucket, gcs_file_path, billing_account_id, default_pubsub_topic, mysql_host, mysql_port, mysql_user, mysql_pass, mysql_db, local_mode, dry_run):
+    setup_logging(APP)
+    setup_metrics(APP)
+
+    # Load config.yaml file
+    if local_mode:
+        with open("config.yaml", "r") as f:
+            budget_dict = yaml.load(f, Loader=yaml.SafeLoader)
+    else:
+        # Load yaml configuration file from GCS
+        gcs_client = storage.Client()
+        bucket = gcs_client.bucket(gcs_bucket)
+        blob = bucket.blob(gcs_file_path)
+        blob.download_to_filename('/app/gcs_budget_config.yaml')
```

Comment:
Instead of writing this object to disk you could instead `download_as_string` and read it in with `yaml.safe_load`

Reply:
done. thanks again